plugins/modules/grpc_get.py

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright 2022 Red Hat
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+
+DOCUMENTATION = """
+module: grpc_get
+version_added: "3.0.0"
+author:
+    - "Ganesh Nalawade (@ganeshrn)"
+    - "Gomathi Selvi S (@GomathiselviS)"
+short_description: Fetch configuration/state data from gRPC enabled target hosts.
+description:
+    - gRPC is a high performance, open-source universal RPC framework.
+    - This module allows the user to fetch configuration and state data from gRPC
+      enabled devices.
+options:
+  section:
+    description:
+      - This option specifies the string which acts as a filter to restrict the portions of
+        the data to be retrieved from the target host device. If this option is not specified the entire
+        configuration or state data is returned in response provided it is supported by target host.
+    aliases:
+    - filter
+    type: str
+  command:
+    description:
+      - The option specifies the command to be executed on the target host and return the response
+        in result. This option is supported if the gRPC target host supports executing CLI command
+        over the gRPC connection.
+    type: str
+  display:
+    description:
+      - Encoding scheme to use when serializing output from the device. The encoding scheme
+        value depends on the capability of the gRPC server running on the target host.
+        The values can be I(json), I(text) etc.
+    type: str
+  data_type:
+    description:
+      - The type of data that should be fetched from the target host. The value depends on the
+        capability of the gRPC server running on target host. The values can be I(config), I(oper)
+        etc. based on what is supported by the gRPC server. By default it will return both configuration
+        and operational state data in response.
+    type: str
+requirements:
+  - grpcio
+  - protobuf
+notes:
+  - This module requires the gRPC system service be enabled on
+    the target host being managed.
+  - This module supports the use of connection=ansible.netcommon.grpc.
+  - This module requires the value of 'ansible_network_os' configuration option (referansible.netcommon.grpc
```

Comment:
can be updated to reflect `grpc_type`
`(referansible.netcommon.grpc connection plugin documentation)` with `(refer ansible.netcommon.grpc connection plugin documentation)`